pubspec.yaml

```diff
@@ -21,7 +21,7 @@ dev_dependencies:
   build_runner: any
   build_web_compilers: any
   collection: ^1.14.10
-  front_end: ^0.1.3
+  front_end: ^0.1.6
```

Comment:
We have a dev dependency on `package: front_end`?

Reply:
Yes, but we shouldn't.  Fixed.